frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -183,38 +183,47 @@ const Settings: NextPage = () => {
-                        <button
-                          type="button"
-                          className={styles["copy-button"]}
-                          title={l10n.getString("settings-button-copy")}
-                          onClick={copyApiKeyToClipboard}
-                        >
-                          <img
-                            src={copyIcon.src}
-                            alt={l10n.getString("settings-button-copy")}
-                            className={styles["copy-icon"]}
-                          />
-                        </button>
-                        <span
-                          aria-hidden={!justCopiedApiKey}
-                          className={`${styles["copied-confirmation"]} ${
-                            justCopiedApiKey ? styles["is-shown"] : ""
-                          }`}
-                        >
-                          {l10n.getString("setting-api-key-copied")}
+                    <div className={styles["settings-api-key-wrapper"]}>
+                      <input
+                        id="api-key"
+                        ref={apiKeyElementRef}
+                        className={styles["copy-api-key-display"]}
+                        value={profile.api_token}
+                        size={profile.api_token.length}
+                        readOnly={true}
+                      />
+                      <span className={styles["copy-controls"]}>
+                        <span className={styles["copy-button-wrapper"]}>
+                          <button
+                            type="button"
+                            className={styles["copy-button"]}
+                            title={l10n.getString("settings-button-copy")}
+                            onClick={copyApiKeyToClipboard}
+                          >
+                            <img
+                              src={copyIcon.src}
+                              alt={l10n.getString("settings-button-copy")}
+                              className={styles["copy-icon"]}
+                            />
+                          </button>
+                          <span
+                            aria-hidden={!justCopiedApiKey}
+                            className={`${styles["copied-confirmation"]} ${
+                              justCopiedApiKey ? styles["is-shown"] : ""
+                            }`}
+                          >
+                            {l10n.getString("setting-api-key-copied")}
+                          </span>
                         </span>
                       </span>
-                    </span>
+                    </div>
+                    <div className={styles["settings-api-key-copy"]}>
+                      {l10n.getString("settings-api-key-description")}
+                      <b>
+                        {" "}
+                        {l10n.getString("settings-api-key-description-bolded")}
+                      </b>
+                    </div>
```

Comment:
I think this probably isn't problematic, but you may want to check with Peying whether she expects the order of these two sentences to change in any language.

If so, you can instead allow the localisers to use the `<b>` tag like this:

```suggestion
                    <Localized id="settings-api-key-description" elems={{ b: <b/> }}>
                      <div className={styles["settings-api-key-copy"]} />
                    </Localized>
```

(see other instances of `<Localized>` for an example.)

Reply:
Yep, I'll definitely defer to her then!

Reply:
Oh, although I see that she just said that it should be in bold, not that it should be separated...

Reply:
Peiying confirmed that this should be fine.